### PR TITLE
Feature/tipspage

### PR DESCRIPTION
**Pull request by marcos-de-barros-vioque**
Hi!
This is the PR for the tips page, including:
- New `tip` and `tipslist` components
- `DB` with content for tips cards
- Checkboxes with filter logic on the tips page, so that the visitor can filter for tips belonging to the desired category
Thanks a lot in advance for checking this!

### CODE REVIEW

File: pages/tips/index.js [hunk 1]
```diff
@@ -1,24 +1,166 @@
...
+            return (
+              // eslint-disable-next-line react/jsx-key
+              <ListItem key={filteredTip.id}>
+                <Tip tips={tip} tip={filteredTip} />
+              </ListItem>
+            );
+          })}
+        </StyledBody>
+      </div>
+    </>
```
Comment (stefankorder) [inline pages/tips/index.js:120]: I would suggest you the following:
- In this case its better to use only the fragment `<>` and no extra `<div>`
- Don't try to disable eslint.
- Do you really need an extra `div` around every `ínput`
- Better write the values of your inputs in lower case.

```suggestion
    <>
        <StyledHeader>
          <StyledTitle>Best Tips & Tricks</StyledTitle>
        </StyledHeader>
        <StyledBody>
          <h3>Choose a category to get the corresponding tips & tricks:</h3>
          <StyledForm>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Food"
                id="essen"
                value="food"
                checked={selectedCategories.includes("Food")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="essen">Food</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Clothes"
                id="kleidung"
                value="clothes"
                checked={selectedCategories.includes("Clothes")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="Kleidung">Clothes</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Household"
                id="haushalt"
                value="household"
                checked={selectedCategories.includes("Household")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="haushalt">Household</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Entertainment"
                id="unterhaltung"
                value="entertainment"
                checked={selectedCategories.includes("Entertainment")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="unterhaltung">Entertainment</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Gasoline"
                id="benzin"
                value="gasoline"
                checked={selectedCategories.includes("Gasoline")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="benzin">Gasoline</StyledLabel>
            </StyledDiv>
            <StyledDiv>
              <StyledCheckboxInput
                type="checkbox"
                name="Restaurants"
                id="restaurants"
                value="restaurants"
                checked={selectedCategories.includes("Restaurants")}
                onChange={handleToggleFilter}
              />
              <StyledLabel htmlFor="restaurants">Restaurants</StyledLabel>
            </StyledDiv>
          </StyledForm>
          {filteredTips.map(filteredTip => {
            return (
              <ListItem key={filteredTip.id}>
                <Tip tips={tip} tip={filteredTip} />
              </ListItem>
            );
          })}
        </StyledBody>
    </>
```

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Take a look at my comments.

File: pages/tips/index.js (hunk 1 above)
Comment (marcos-de-barros-vioque) [reply pages/tips/index.js:120]: I see, thanks @stefankorder ! I'm updating this PR according to your comments.

**Comment by marcos-de-barros-vioque** [comment]
@stefankorder 

I've updated the PR according to your comments.

With regards to the following points: 

>Do you really need an extra div around every ínput

The reason why I am doing it is to have the pairs (input+label) in 2 columns. So far, I couldn't come with a better solution to resolve this.

>Better write the values of your inputs in lower case.

I've tried to change the values to lower case (and also in the db.json accordingly). However, the filtering doesn't work properly (no tips are being displayed by selecting any of the options).

**Review by stefankorder — APPROVED** [review]
LGTM!

**Pull request merged**